### PR TITLE
Prepare release v1.29.0

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
@albertteoh, as the previous release manager, would you mind reviewing this one?

File: RELEASE.md [hunk 1]
```diff
@@ -42,5 +42,6 @@ Here are the release managers for future versions with the tentative release dat
 | 1.30.0    | @pavolloffay     |  5 January   2022      |
 | 1.31.0    | @vprithvi        |  2 February  2022      |
 | 1.32.0    | @yurishkuro      |  2 March     2022      |
-| 1.33.0    | @joe-elliott     |  6 April     2022      |
+| 1.33.0    | @jpkrohling      |  6 April     2022      |
```
Comment (jpkrohling) [inline RELEASE.md:45]: I'm swapping releases with @joe-elliott.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,30 @@ next release
...
+-------------------
+### Backend Changes
+
+#### Breaking Changes
+
 * Remove deprecated `--badger.truncate` CLI flag in [#3410](https://github.com/jaegertracing/jaeger/pull/3410))
 
 #### New Features
 
+* Expose rackID option in ingester [#3395](https://github.com/jaegertracing/jaeger/pull/3395))
```
Comment (pavolloffay) [inline CHANGELOG.md:21]: the format of items does not follow the previous format (author, PR).  Also the breaking change item on line 17 should be changed

**Review by pavolloffay — COMMENTED** [review]
Apart from items formatting it looks good

File: CHANGELOG.md (hunk 2 above)
Comment (jpkrohling) [reply CHANGELOG.md]: PR updated, but not sure what you meant regarding line 17. Does it look OK now, or should it still be changed?

Comment (pavolloffay) [reply CHANGELOG.md]: it looks fine now. It didn't look good previously so I explicitly mentioned it as it was not added in this PR

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**